### PR TITLE
Check items on closing issues

### PR DESCRIPTION
**Pull request by Spredzy**
This PR contains two commits.

The first one is to integrate new facts (github_issue and github_closed_issue).
The second one is to take automatic action of closing trello checklists items when an issue that was listed is closed.

### CODE REVIEW

File: github.pl [hunk 1]
```diff
@@ -22,10 +22,17 @@
...
+    config(github, [Owner, Project]),
+    github_issues(Owner, Project, Issues),
+    member(Issue, Issues),
+    store_fact(Gen, github_issue(Owner, Project, Issue.id, Issue.html_url, Issue.title, Issue.state, Issue.user.login)).
+
 merge_status(false, no).
 merge_status(true, yes).
 
 :- add_fact_updater(github:update_github_facts).
+:- add_fact_updater(github:update_github_issues_facts).
```
Comment (fredericlepied) [inline github.pl]: why don't you call it github:update_github_facts ?

File: githublib.pl [hunk 2]
```diff
@@ -24,4 +24,8 @@
     format(atom(Path), '/repos/~w/~w/pulls', [Owner, Project]),
     github_json(Path, Dict).
 
+github_issues(Owner, Project, Dict) :-
+    format(atom(Path), '/repos/~w/~w/issues?state=all&access_token=e427390592197574257a31f6e8a81cad8697df41', [Owner, Project]),
```
Comment (fredericlepied) [inline githublib.pl]: what is this token?

**Review by fredericlepied — CHANGES_REQUESTED** [review]

File: github.pl (hunk 1 above)
Comment (Spredzy) [reply github.pl]: done.

File: githublib.pl (hunk 2 above)
Comment (Spredzy) [reply githublib.pl]: Nothing, act like you've never seen it :)

**Pull request merged**